www/pfSense-pkg-Lightsquid/files/usr/local/pkg/lightsquid.inc

```diff
@@ -328,12 +335,16 @@ ssl.honor-cipher-order	= "enable"
 ssl.cipher-list		= "AES128+EECDH:AES256+EECDH:AES128+EDH:AES256+EDH:AES128-SHA:AES256-SHA:!aNULL:!eNULL:!DSS"
 EOF;
 
-		// Configure IPv4 and IPv6 sockets
+		/*
+		 * IPv4 HTTPS bind must NOT be a socket; otherwise the retarded
+		 * lighttpd thing yet again binds to 80/HTTP no matter what.
+		 * You enable SSL-only IPv4 by not configuring bind for IPv4 at all.
+		 * IPv6 SSL must be configured as a socket
+		 */
 		$lighttpd_ls_conf .= <<<EOF
 
```

Comment:
OK, it's really good to trigger some upstream response here, thanks. 

- My expectation is that if I don't configure ```server.bind``` well that should mean really any, and dual-stack if available, no? As in "I don't want to restrict the bind in any specific way, why configure it then? If I wanted, I'd put it into the config". The wiki somehow suggests that should be the case, but not really in the end. It just doesn't work that way. Not for the use case here, and not for many others.
- The "user-friendly" part cost me a lot of hair and two keyboards. Document how to skip it. Yeah, the ```server.port``` thing does avoid it, but that info is nowhere to be found.
- And when you've somehow managed that, you are still in the protocols mixup pitfall with dual stack. (As you've already acknowledged, the global ssl config is not really respected, so you end up with those repeated blocks of redundant code.)

P.S. As for backward compatibility - well yeah if you break the code we have here in future, I'll scream and vent here again. :rofl: